web/src/utils/uuidToURL.ts

```diff
@@ -0,0 +1,20 @@
+export const uuidToURL = (uuid: string, objectName: string) => {
+  // TODO: needs proper implementation
+  if (!uuid || !objectName) return "";
+  const storageHost = window.REEARTH_CONFIG?.storageHost ?? "http://localhost:8080";
```

Comment:
The `storageHost` in the config currently if empty falls back to `"/"`. If it should be localhost:8080 (or "/" doesn't resolve to that), please set the env variable there. 
I think its better if you leave the env variables to the config and just if there is no storageHost return.

Reply:
Done!